### ISSUE TITLE
Resume a deposit

### ISSUE THREAD

**Issue by pdyraga** [threshold-network/token-dashboard#457]
#456 will open the deposit flow at the right step but it will not handle revealing the deposit. When someone downloaded JSON but did not reveal it, we need a UI to upload that JSON and resume the deposit.

For this issue we need to:
- [x] Design the resume from the JSON.
- [x] Implement it!

**Comment by sashatanase** [comment]
It was mentioned at some point, minting in parallel from the same address. Is this still a thing? @mhluongo said it yesterday, and I was wondering how someone can mint in parallel with the same wallet address. 

I might've misheard it, but I want to ensure I have valid data.

**Comment by pdyraga** [comment]
For now, this is just about having a unique URL for a deposit. Allowing for multiple mints simultaneously requires more work - we need to check what data we store in the browser's storage. We'll have a separate issue for that.

**Comment by mhluongo** [comment]
Confirming though that it is possible @sashatanase — I can stagger 10 deposits at once if I want. Our dApp won't support it, but I could do it with multiple browser profiles or by using EtherScan directly.

There's not much reason to do this as a user if you plan ahead, but I already did it once on accident to save time :see_no_evil:

**Comment by sashatanase** [comment]
**Context** 

We need to provide the users with a way of resuming their minting. I assume that most cases with the revealing issues will be due to gas errors triggered by low funds or gas set too low. 


**Solution**

In the event of a transaction error as soon as the user hits `Dismiss` they will be shown the Mint Resuming page inside of the dApp



![Image](https://user-images.githubusercontent.com/40768736/222213557-92a752bf-9657-4080-ae75-9dfdf0376aee.png)



![Image](https://user-images.githubusercontent.com/40768736/222213654-522ca75f-7506-4939-83d9-d81685136330.png)

As soon as they Upload  and Click Resume this is what they'll see


![Image](https://user-images.githubusercontent.com/40768736/222213819-1b3d999c-1f6b-4964-b03f-cac3b028ae2f.png)



![Image](https://user-images.githubusercontent.com/40768736/222213909-f2e1e43f-b712-4dc5-b6e4-94e484408ff5.png)

We will also show in the My Activity Card that they have a mint that needs to be resumed.



![Image](https://user-images.githubusercontent.com/40768736/222214222-4cef207c-229d-4489-82cf-87805228f121.png)



In case the user's browser closes due to various reasons as soon as they open the dApp in the browser and connect their wallet this is what they see 



![Image](https://user-images.githubusercontent.com/40768736/222215142-21f2ab9b-140a-4334-b95f-585d50a9a4c9.png)

In case they click dismiss, we have a measure of error prevention through the dApp

First one is an error prevention modal in case they click `Dismiss`in the previous modal. 


![Image](https://user-images.githubusercontent.com/40768736/222215763-0e2f6dd4-78d5-4bd8-ba37-d6b6266f3a7d.png)

We will also show the fact that they need to reveal on the Overview page

![Image](https://user-images.githubusercontent.com/40768736/222216003-be752682-13b6-4576-8627-2b36c1ffc8e1.png)

And we'll make sure they see that they need to reveal even if they dismissed the modals everywhere. 



![Image](https://user-images.githubusercontent.com/40768736/222216794-b08adcd0-6e2b-4a1a-876b-36daa15a6c1c.png)

@pdyraga Let me know if this works

**Comment by pdyraga** [comment]
Unassigning @sashatanase as it is queued for implementation now.

**Comment by sashatanase** [comment]
@r-czajkowski Not every wallet has stakes ... I believe that the tBTC bridge user will be entirely separate from the stakers, so I do not see how exactly they would be displayed in the authorize staking apps and operator mapping.
